android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlService.java

```diff
@@ -450,4 +531,97 @@ public void onError(String error) {
 			sdlManager.getScreenManager().presentChoiceSet(choiceSet, InteractionMode.MANUAL_ONLY);
 		}
 	}
+
+	public static class UIStreamingDisplay extends SdlRemoteDisplay {
```

Comment:
@santhanamk this change is related more to demonstrate proposal implementation, rather than to a functional one. If it is required, I can separate this code, but, I believe, this should not go to the final release version and should be deleted. Though, it remains here right now in order to help everybody to acknowledge. Should I remove it now?